### ISSUE TITLE
Flushing EOF when running docker-compose for the first time

### ISSUE THREAD

**Issue by CXwudi** [HelloGrasscutter/HelloGrasscutter#2]
This happens to me using the default `docker-compose.yml` file provided in the repo  
I am using Win10 Docker with WSL2
```
> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.

> [12:20:38] [INFO] EOF detected.
```

**Comment by Becods** [comment]
Maybe it's a Grasscutter error.
Try to start Grasscutter in wsl without docker.

**Comment by CXwudi** [comment]
I don't think this is related to an environment problem. It might be caused by `nohup` in https://github.com/MuirProject/HelloGrasscutter/blob/402b0a2336bd246e5f1c24570922b5b29fc05c94/development/rootfs/entrypoint.sh#L8  
[This issue](https://github.com/Grasscutters/Grasscutter/issues/508) has pointed out that removing `nohup` seems to solve this `EOF detected` log flushing problem. 

Btw, I suspected there is nothing for Grasscutter dev to do with this issue because I suspected the log came from netty itself, as the source code of Grasscutter doesn't contain a string of `"EOF detected"`

**Comment by Becods** [comment]
This nohup is used to generate config.json, which will force the process to end after 3 seconds, but I don't know why the process is not ended, I need some time to look into it.

I have updated the code I am trying to fix in the [fix-#2](https://github.com/MuirProject/HelloGrasscutter/tree/fix-%232) branch and uploaded the [docker image](https://hub.docker.com/layers/209789404/becod/hellograsscutter/fix-2/images/sha256-e86c8a64c53a33c90427970b10458bd198282bf15fb1eeb4b8dca2f5612f90f3?context=repo). Since I can't reproduce the configuration environment, I hope you can help me test.

**Comment by CXwudi** [comment]
Yes, it works, although I got another exception but I think that is just a minor issue, no need to bother.
```
......
[17:14:32] [INFO] Quartz scheduler version: 2.3.2
[17:14:32] [INFO] Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED started.
[17:14:32] [INFO] Drop data successfully loaded.
[17:14:32] [INFO] Expedition reward successfully loaded.
[17:14:32] [INFO] Logging initialized @2444ms to org.eclipse.jetty.util.log.Slf4jLog
The following languages mappings are available, please select one: [default: EN]
>   en  id  fr  cht vi  chs 
>   es  jp  kr  de  ru  pt  
>   th  

Exception in thread "main" org.jline.reader.EndOfFileException
	at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:653)
	at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:468)
	at emu.grasscutter.tools.Tools.getLanguageOption(Tools.java:86)
	at emu.grasscutter.tools.Tools.createGachaMapping(Tools.java:40)
	at emu.grasscutter.server.dispatch.DispatchServer.start(DispatchServer.java:457)
	at emu.grasscutter.Grasscutter.main(Grasscutter.java:109)
```

Anyways, thanks for the fix
